fetch.bs

```diff
@@ -362,6 +348,8 @@ a <a for=/>header list</a> (<var>list</var>), run these steps:
    <a for=header>combined value</a> given <var>name</var> and
    <var>list</var>.
 
+   <li><p>Set <var>name</var> to <var>name</var>, <a>byte-lowercased</a>.
```

Comment:
Shouldn't we lowercase _name_ before sorting? Imagine _headers_ contains "a", "B", "c" and "D" as names. _names_ will be ["B", "D", "a", "c"] and after lowercased it will be ["b", "d", "a", "c"].